static/app/views/performance/vitalDetail/index.tsx

```diff
@@ -26,15 +25,13 @@ import {addRoutePerformanceContext, getTransactionName} from '../utils';
 
 import VitalDetailContent from './vitalDetailContent';
 
-type Props = {
+type Props = RouteComponentProps<{}, {}> & {
   api: Client;
   location: Location;
```

Comment:
should we remove location and maybe use Pick here too? :D

```suggestion
```

Reply:
Removed `location` 👍